integrations-and-supported-tools/pytorch/scripts/helpers.py

```diff
@@ -0,0 +1,46 @@
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+from neptune.new.types import File
+import torch.nn.functional as F
+
+# Helper functions
+def save_model(model, name ="model.txt"):
+    print(f"Saving model arch as {name}.txt")
+    with open(f"{name}_arch.txt", "w") as f:  f.write(str(model))
+    print(f"Saving model weights as {name}.pth")
+    torch.save(model.state_dict(), f"./{name}.pth")
+    
+def get_obj_name(obj):
+    return type(obj).__name__
+
+def save_image_predictions(model, validloader, run, n_samples = 50):
+    classes = ["airplane", "automobile", "bird", "cat", "deer", "dog", "frog","horse","ship","truck"]
+    dataiter = iter(validloader)
+    images, labels = dataiter.next()
+    model.eval()
+
+    # moving model to cpu for inference 
+    if torch.cuda.is_available(): model.to("cpu")
+
+    # predict batch of n_samples
+    img = images[:n_samples]
+    probs = F.softmax(model(img),dim=1)
+    probs = probs.data.numpy()
+
+    # Decode probs and Log images
+    for i, ps in enumerate(probs):
+        pred = classes[np.argmax(ps)]
+        gt = classes[labels[i]]
+        description = "\n".join(
+            ["class {}: {}%".format(classes[n], round(p*100, 2)) for n, p in enumerate(ps)]
+        )
+        # Log Series of Tensors as Image and Predictions. For more see ->
```

Comment:
I don't think we should link to docs from here -> it's an example and we should try and focus on example rather than education of other docs pages I think

Reply:
I thought of this because helpers is hidden and specific for more options so if someone goes there they might one to understand a little bit more.